settings.gradle

```diff
@@ -26,63 +26,44 @@ plugins {
 }
 
 rootProject.name = "opentelemetry-java"
-include ":opentelemetry-all",
-        ":opentelemetry-api",
-        ":opentelemetry-api-baggage",
-        ":opentelemetry-api-common",
-        ":opentelemetry-api-trace",
-        ":opentelemetry-api-metrics",
-        ":opentelemetry-context",
-        ":opentelemetry-extension-annotations",
-        ":opentelemetry-extension-kotlin",
-        ":opentelemetry-extension-trace-propagators",
-        ":opentelemetry-exporter-jaeger",
-        ":opentelemetry-exporter-jaeger-thrift",
-        ":opentelemetry-exporter-logging",
-        ":opentelemetry-exporter-otlp",
-        ":opentelemetry-exporter-otlp-metrics",
-        ":opentelemetry-exporter-otlp-trace",
-        ":opentelemetry-exporter-prometheus",
-        ":opentelemetry-exporter-zipkin",
-        ":opentelemetry-integration-tests",
-        ":opentelemetry-integration-tests-tracecontext",
-        ":opentelemetry-opencensus-shim",
-        ":opentelemetry-opentracing-shim",
-        ":opentelemetry-proto",
-        ":opentelemetry-sdk-common",
-        ":opentelemetry-sdk-metrics",
-        ":opentelemetry-sdk-trace",
-        ":opentelemetry-sdk",
-        ":opentelemetry-sdk-extension-async-processor",
-        ":opentelemetry-sdk-extension-aws",
-        ":opentelemetry-sdk-extension-logging",
-        ":opentelemetry-sdk-extension-otproto",
-        ":opentelemetry-sdk-extension-resources",
-        ":opentelemetry-sdk-extension-tracing-incubator",
-        ":opentelemetry-sdk-extension-jaeger-remote-sampler",
-        ":opentelemetry-sdk-extension-jfr-events",
-        ":opentelemetry-sdk-extension-zpages",
-        ":opentelemetry-sdk-testing",
-        ":opentelemetry-bom",
-        ":opentelemetry-perf-harness"
-
-rootProject.children.each {
```

Comment:
I would be happy to get rid of this, but the archiveFilename workaround does not look much better all-in-all IMHO. Maybe we can rename the directories so we can get rid of that workaround too?

Reply:
Ah my comment might have been confusing, the workaround is only setting the group so group + name are unique.

The rest is applying our convention for archive names in a generic way. Another alternative is to not apply rules and just set `archivesBaseName` in each Gradle file. It's what I do for my own projects. Don't have a particular preference for either though.

I do like the current folder names since the gradle commands are much shorter and easier to execute. Gradle zsh completion has never worked well for me so always have to type everything out.

Reply:
Oh, I see. So basically we are doing the same with the archiveBasenames now that we used to to with the project directory names before, but distributed over multiple files.

I think I would prefer either having everything in one place (maybe the top-level build.gradle) or, like you said, have each project's build.gradle just set it's own name. As it is now in this PR, it's kind of a mixture of the two.